CHANGELOG.md

```diff
@@ -79,13 +89,13 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 - replace term tracker item with term tracker annotation (#1922, #1923)
 - replace has bearer with characteristic of (#1928)
 
-### Obsoletion
+### Removed
```

Comment:
Actually, this was an obsoletion, not a removal. I can reverse this in another PR.

Reply:
I have never seen this section before and don't understand the use here.
For the changelog it is not a common section.

Was it removed from the repo? -> removed
Is it still part of the repo? -> updated

Reply:
Would "Deprecated" be a nice compromise?